index.html

```diff
@@ -523,8 +523,18 @@ <h2>
 
     <section>
       <h2>
-Keys
+Cryptographic Keys
       </h2>
+      <p>
+A DID Document can contain cryptographic keys and other material that can be
+used to authenticate or authorize interactions with the subject of the DID or
+associated parties. These keys often contain identifiers that allow them to be
```

Comment:
```suggestion
associated parties. The information expressed about keys often includes globally unambiguous identifiers and public key material which can be used to verify digital signatures.
```

Above suggestion might need to be manually added -- github isn't friendly with EOLs yet.

Reply:
Fixed.